docs/routing/special_routes.md

```diff
@@ -0,0 +1,56 @@
+# Error handling in router
+
+Сurrently ILC can only handle [404](#404-error) and [5XX](#5xx-error) errors.
+
+Our web page now is composed from several fragments, that's why we can't use the same approach we used with monolithic frontend.
+This document describes how these errors were realized in ILC and why in this way.
+
+## '404' Error
+
+To see 404 error routes, use the `Show special` switcher at the top of the routes page. `Special route` must be created separately for each domain.
+
+More about [404 Error here](../global_errors_handling.md#404-error-not-found).
+
+![ILC registry special routes switcher](../assets/routes/special-routes-switcher.png)
+
+Unlike simple routes which use composition with the help of `Next: true|false` property, the special route has own full configuration. This means that you must add template and all the necessary applications (header, footer, etc.) to the `Special route` [slot](./route_configuration_options.md#slot-configuration) (as in the screenshot below).
+
+![ILC registry error slot](../assets/routes/error-slot.png)
+
+### Example:
```

Comment:
Replace this section with a link to https://github.com/namecheap/ilc/blob/e6e5809f2d9e90de1db3c3b4ff5bce8a37fdf4d1/docs/global_errors_handling.md#404-error-not-found